client/src/components/Browse/Browse.js

```diff
@@ -1,65 +1,61 @@
 import React from "react";
 import "../../db/Browse";
-import { BrowseContainer, BackgroundImage, TitleDiv, CustomTitle, CustomTitleWithCarrot } from "./Browse.styles";
-import './Browse.styles'
-
+import {
+  BrowseContainer,
+  BackgroundImage,
+  TitleDiv,
+  CustomTitle,
+  CustomTitleWithCarrot,
+  BrowseBtn,
+  ButtonContainer,
+  ByTitle,
+} from "./Browse.styles";
+import "./Browse.styles";
+import { browse_platform, browse_genre, browse_theme, browse_keyword } from "../../db/Browse";
 
 const Browse = () => {
-   function makePlatformBtn(){
-     return(
-       
-     )
-   }
-
-        return (
-          <BrowseContainer>
-            <BackgroundImage/>
-            <TitleDiv>
-              Browse 
-              <CustomTitle>customize</CustomTitle>
-              <CustomTitleWithCarrot>Customize&gt;</CustomTitleWithCarrot>
-            </TitleDiv>
-
-          </BrowseContainer>
-        
+  function makePlatformBtn(browse_platform) {
+    return <BrowseBtn>{browse_platform.platform}</BrowseBtn>;
+  }
+
+  function makeGenreBtn(browse_genre) {
+    return <BrowseBtn>{browse_genre.genre}</BrowseBtn>;
+  }
+
+  function makeThemeBtn(browse_theme) {
+    return <BrowseBtn>{browse_theme.theme}</BrowseBtn>;
+  }
+
+  function makeKeywordBtn(browse_keyword) {
+    return <BrowseBtn>{browse_keyword.keyword}</BrowseBtn>;
+  }
+
+  return (
+    <BrowseContainer>
+      <BackgroundImage />
+      <TitleDiv>
+        Browse
+        <CustomTitle>customize</CustomTitle>
+        <CustomTitleWithCarrot>Customize&gt;</CustomTitleWithCarrot>
+      </TitleDiv>
+      <ButtonContainer>
```

Comment:
ButtonContainer should Ideally be an unordered list instead of a div and each item should be an `li` with a `button` element inside of it.  ButtonContainer should also be named something more like `BrowsePageSection` or somthing similar.  It should be a section that has a `SectionTitle` and then all of the element.  

Also, no need for the this keyword that is unnecessary.